pkg/apis/app/v2/kieapp_types.go

```diff
@@ -977,6 +979,20 @@ type KafkaExtObject struct {
 	MaxBlockMs *int32 `json:"maxBlockMs,omitempty"`
 }
 
+// KafkaJBPMEventEmittersObject kafka configuration to be used by the KieApp for JBPM Emitter
+type KafkaJBPMEventEmittersObject struct {
+	// Jbpm emitter, comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster.
```

Comment:
remove the jbpm emitter string from its description.
And seems it still not aligned with https://github.com/jboss-container-images/rhpam-7-openshift-image/pull/550/files

Reply:
ok